.github/workflows/ci-cd.yml

```diff
@@ -46,8 +46,14 @@ jobs:
           yarn nx affected --target=lint --base=${{ env.BASE_BRANCH }} --parallel=3
       - name: build & test
         run: |
-          eval "yarn nx affected --target=build --base=${{ env.BASE_BRANCH }} --parallel=3"
           eval "yarn nx affected --target=e2e --base=${{ env.BASE_BRANCH }} --parallel=1 --skip-nx-cache"
+          rm -rf dist
+          if [[ $BRANCH_REF != 'refs/heads/main' ]]; then
```

Comment:
Do you need to restore the base URL for the main branch?

Reply:
For main branch, we will use the default base URL set in the secrets/environment variables